rest/admin_api_test.go

```diff
@@ -565,6 +565,26 @@ func TestLoggingCombined(t *testing.T) {
 	goassert.DeepEquals(t, logKeys, map[string]bool{"Changes": true, "Cache": true, "HTTP": true})
 }
 
+func TestGetStatus(t *testing.T) {
+	rtConfig := RestTesterConfig{NoFlush: true}
+	rt := NewRestTester(t, &rtConfig)
+	defer rt.Close()
+
+	response := rt.SendRequest("GET", "/_status", "")
+	assertStatus(t, response, 404)
+
+	response = rt.SendAdminRequest("GET", "/_status", "")
+	assertStatus(t, response, 200)
+	var responseBody Status
+	json.Unmarshal(response.Body.Bytes(), &responseBody)
```

Comment:
We're not checking this response is unmarshalled into the struct successfully (missing error check)